### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by xjlu**
1. rake task accepts a single source string 
2. improve and clarify readme for usage in a rails project

### CODE REVIEW

File: lib/double_doc/task.rb [hunk 1]
```diff
@@ -8,12 +8,13 @@ module DoubleDoc
...
   ## It is very easy to set up a rake task for generating your documentation. All you have to do is
-  ## tell DoubleDoc what the input files are, and where you want the output to go.
+  ## tell DoubleDoc what the input files are, and where you want the output to go. In the example,
+  ## `zdoc` is picked to avoid conflicts with rails doc rake tasks.
   ##
   ## ```ruby
   ## require 'double_doc'
   ##
-  ## DoubleDoc::Task.new(:doc,
+  ## DoubleDoc::Task.new(:zdoc,
```
Comment (staugaard) [inline lib/double_doc/task.rb]: Why `zdoc`?

Comment (staugaard) [inline lib/double_doc/task.rb]: please call the task `double_doc` or something.

Comment (xjlu) [inline lib/double_doc/task.rb]: Changed to `double_doc`.

**Comment by staugaard** [comment]
:+1:

**Pull request merged**